shared/p2p/service.go

```
@@ -101,6 +101,7 @@ func checkAvailablePort(port int) bool {
 		return false
 	}
 
+	// #nosec G104
```

Why are we ignoring this error? Please handle the error.

---

Alright handled the error